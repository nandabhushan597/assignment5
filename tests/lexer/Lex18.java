"this is a string"
  "this is another"
  "this string has a \" in it"
  "This \" one \" has \" several"
  "\b\t\\"
  "\\\\"
  "\\77\77"  
