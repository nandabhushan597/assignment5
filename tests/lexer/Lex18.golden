Right [LiteralT (StringL "this is a string")
LiteralT (StringL "this is another")
LiteralT (StringL "this string has a \" in it")
LiteralT (StringL "This \" one \" has \" several")
LiteralT (StringL "\b\t\\")
LiteralT (StringL "\\\\")
LiteralT (StringL "\\77?")]